invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -42,6 +42,30 @@
     </div>
   </div>
   {% endif %}
+  {% if is_preview %}
+  <div class="ui info flashed top-attached manage message">
+    <div class="ui container">
+      <div class="ui relaxed grid">
+        <div class="column">
+          <div class="row">
+            <div class="header">
+              <i class="eye icon"></i> Preview
+            </div>
+            <p>
+              {% if record.is_published %}
+                You are previewing changes that has not yet been published.
+              {% endif %}
+
+              {% if not record.is_published and record.versions.index >= 1 %}
```

Comment:
Minor: The third case is missing here - ``if not record.is_published and record.versions.index == 1``